packages/api/src/routes/medical/internal-docs.ts

```diff
@@ -168,6 +174,32 @@ router.post(
         requestId,
         source,
       });
+
+      const externalData =
+        source === MedicalDataSource.COMMONWELL
+          ? await getCWData(updatedPatient.data.externalData)
+          : await getCQData(updatedPatient.data.externalData);
+
+      const isConversionCompleted =
+        externalData?.documentQueryProgress?.convert?.status === "completed";
+
+      if (isConversionCompleted) {
+        const startedAt = externalData?.documentQueryProgress?.startedAt;
+
+        const duration = diffFromNow(startedAt);
+
+        analytics({
```

Comment:
Ditto, try to make non-functional code like analytics less prominent with fewer empty lines?